web-ui/src/main/resources/catalog/components/viewer/owscontext/OwsContextService.js

```diff
@@ -357,6 +359,7 @@
             }
             firstLoad = false;
           }
+          gnViewerService.openTool("layers");
```

Comment:
Maybe make this configurable, to me it seems odd display the layers panel when you access the map viewer.

It seems only happens when loading the map context file the first time, if the context is loaded from cache doesn't seem to happen.

Reply:
Configuration added in https://github.com/geonetwork/core-geonetwork/pull/7126